src/core_modules/capture-core/components/DataEntries/common/useRegistrationFormInfoForSelectedScope.js

```diff
@@ -27,13 +29,14 @@ const useRegistrationOptions = (): RegistrationOptions => {
                     name: trackedEntityTypeName,
                     formId: `registrationPageForm-${trackedEntityTypeId}`,
                 },
-                ...programs.reduce((accumulated, { programId, programName, enrollment }) => ({
+                ...programs.reduce((accumulated, { programId, programName, enrollment, firstStageForm }) => ({
```

Comment:
Same as the comment on `useTrackedEntityTypesWithCorrelatedPrograms`. We're trying to move away from these memoryObjects, and should therefore try to align these PR's.